server/views/pages/search.njk

```diff
@@ -51,21 +49,74 @@
+              <p>Most of the works were acquired between 1890 and 1936 by Sir Henry Wellcome and his employed agents across the globe. The images largely reflect Wellcome’s collecting interests and were designed to form a documentary resource that reflects the cultural and historical contexts of health and medicine.</p>
+              <p>Henry Wellcome had a personal interest in medical and ethnographical objects and the objects, artworks and photographs he collected were initially presented in the Wellcome Historical Medical Museum. Over the subsequent decades the library and its collections developed to become the Wellcome Collection as it is now: a free museum and library exploring health, life and our place in the world.</p>
+              <p>Many of the images on this site were digitised during the 1990sand first made available online in 2002. Recent developments to the site have made these images more easily discoverable, but have also brought to the surface legacy issues such as the sensitive nature some of the content, and the poor quality of some of the early digitisation. As we aim to make more images from the library collections available over the coming months, we have planned a programme of work to enable us to identify and consider these issues in a systematic and consistent manner We would like to include a range a voices from inside and outside of Wellcome to assist us with this. If you would like to get involved and have information about an image which might help us to understand it better, please email <a href="mailto:collections@wellcome.ac.uk">collections@wellcome.ac.uk</a>.</p>
+              <p>Visit the pages below for more information about the history of Sir Henry Wellcome, Wellcome Collection and Wellcome Library.</p>
+            </div>
+
+            <div class="{{ {s:6, l:0} | spacingClasses({margin: ['bottom']}) }}">
+              <div>
+                {% componentV2 'more-info-link', {name: 'History of Wellcome', url: 'https://wellcome.ac.uk/about-us/history-wellcome'} %}
+              </div>
+              <div>
+                {% componentV2 'more-info-link', {name: 'Introducing Henry Wellcome', url: 'https://wellcomecollection.org/what-we-do/introducing-henry-wellcome'} %}
+              </div>
+              <div>
+                {% componentV2 'more-info-link', {name: 'History of Wellcome Library', url: 'https://wellcomelibrary.org/what-we-do/history-of-wellcome-library/'} %}
+              </div>
+            </div>
+          </div>
+          <div class="{{ {s: 12, m: 8, l: 5, xl: 5} | gridClasses }}">
+            {% set henryImage = {
+              type: 'picture',
+              contentUrl: 'https://s3-eu-west-1.amazonaws.com/miro-images-public/V0027000/V0027772.jpg',
```

Comment:
Personal opinion:

If we are looking at diversifying things, I know there is history and heritage here, but we could move away from the dead white man portraiture.

Reply:
@jamesgorrie As much as we want to diversify, the purpose of this copy is to address the complaints about the content of the collection. We are hoping to address it by giving some context and explain how the collection started. Hence the Henry Wellcome portrait and being transparent about the content of the collection. If you have other suggestions, let me know.

Reply:
I love this message, its one of the reasons I work here is that we acknowledge the past as the past, contextual, and not necessarily indicative of the trajectory of the future.

[A great example of this is the returning of Australian Aboriginal artefacts](https://wellcome.ac.uk/funding/managing-grant/return-australian-aboriginal-skulls-wellcome-collections-held-science-museum).

My suggestion is perhaps we could follow the same message by choosing an image that is more representative of what we are and how we would like to be, not so focused on the past. Potentially around health?

I've done a bit of searching, nothing much yet, but will keep trying:
https://wellcomecollection.org/works/tm6fecve
The little figure from here: https://wellcomecollection.org/articles/WcpARygAAOwHETW0